.prettierrc

```diff
@@ -0,0 +1,12 @@
+{
+  "printWidth": 100,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true,
+  "singleQuote": false,
```

Comment:
Hmmm..., do we really want to enforce double quote preference across all files, regardless of whether they are JS or JSX files? It is not much of an issue, but it sort of hit me as something strange, and I wanted to double check.

Reply:
This setting does not apply for JSX files, in that case they will always be double quotes and this option is ignored. Also, prettier is smart enough to handle quotes inside string literals, check [this](https://jsfiddle.net/fjcalzado/wn5xoq6x/1/).

In regards to double vs single quotes .... yes .... I know this may be controversial 😈. Some prefer single, some other go for double (*quote wars*). Honestly I prefer double quotes, but I am not biased, I can change my mind easily if needed. The true reason to go for double is just that most of the style consistency tools I have used (ESLint, Prettier, etc) favour double quotes over single by default. Also, since React came to the scene, many have shifted to double quotes due to the tendency to add attributes between double quotes in JSX (since JSX is closer to HTML). 

But I know Lemoncode guys tend to use single quotes much more so .... 😆  ... this is a lost cause. Let's @brauliodiez decide!

Reply:
Yeah, I had checked some of prettier's functionality after your comment, I did not know about this tool, and seems quite handy.  In short, the questions here is: do we follow the single-quotes convention for JS files, and double-quotes convention for JSX? Or do we instead use the same convention for the whole project (either single or double)?
I think every option here is actually good enough, but as you said, not for us to decide in the end :stuck_out_tongue_closed_eyes: